docs/function-schemas.md

```diff
@@ -479,6 +479,23 @@ with instrumentation:
 ; =throws=> :malli.core/invalid-input {:input [:cat :int [:int {:max 6}]], :args [10 10], :schema [:=> [:cat :int [:int {:max 6}]] :int]}
 ```
 
+... or by using metadata `^:malli/always-check`:
```

Comment:
I would use `:malli/always-validate` (like plumatic) or even `:malli/always` (aesthetics). I like the latter as it's shorter.

Reply:
`:malli/always` is fine by me